models/reviews.js

```diff
@@ -13,3 +13,20 @@ exports.selectReviews = () => {
     return results.rows;
   });
 };
+
+exports.fetchReviewByID = (id) => {
+  return db
+    .query(
+      "SELECT reviews.*, COUNT (comment_id) AS comment_count FROM reviews LEFT JOIN comments ON reviews.review_id = comments.review_id WHERE reviews.review_id = $1 GROUP BY reviews.review_id",

```

Comment:
No requirement for you to be adding comment count when you're getting an individual review, please remove this from the model query.